index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Portfolio:A programmer</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header class="main-header">
+      <h1 class="mh-logo">
+      <a href="https://c.tenor.com/19GG239lY8UAAAAM/excellent-excellent-job.gif" class="logo">
+        Chaudhary
+      </a>
+    </h1>
```

Comment:
- [ ] The logo shouldn't be inside an `h1` tag, kindly change that. 😁
- [ ] Also, please change the styling of the logo as right now it's using the default styling of the `a` element, as well as the `font-family`, it should be `Poppins`.
This is how yours look:
![image](https://user-images.githubusercontent.com/20567503/177659743-15b28f9c-6472-4146-838f-6e87c778af70.png)
This is how it should look, following the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1):
![image](https://user-images.githubusercontent.com/20567503/177659777-f7142166-a3e5-4d30-9a79-349a62f22b99.png)